test/integration_test/specs/sysbench-backup/applicationbackup.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: stork.libopenstorage.org/v1alpha1
+kind: ApplicationBackup
+metadata:
+  name: sysbench-backup
```

Comment:
Why would you want to backup the sysbench app? You should run sysbench before the backup, do a backup+restore and then verify that the data written by sysbench matches

Reply:
sysbench app actually contains two containers, one is mysql and the second is mysql where sysbench generates load.

Reply:
got it, now sysbench is removed. this PR will just add parametrization for running app.